authorizations/authorization-ranger/src/main/java/org/apache/gravitino/authorization/ranger/RangerAuthorizationHDFSPlugin.java

```diff
@@ -679,4 +680,57 @@ public Boolean onMetadataUpdated(MetadataObjectChange... changes) throws Runtime
     }
     return Boolean.TRUE;
   }
+
+  @Override
+  protected String getServiceType() {
+    return "hdfs";
+  }
+
+  @Override
+  protected Map<String, String> getServiceConfigs(Map<String, String> config) {
+    String usernameKey = "username";
+    String usernameVal = "admin";
+    if (config.containsKey(BaseCatalog.CATALOG_BYPASS_PREFIX + usernameKey)) {
+      usernameVal = config.get(BaseCatalog.CATALOG_BYPASS_PREFIX + usernameKey);
+    }
+
+    String passwordKey = "password";
+    String passwordVal = "admin";
```

Comment:
This .... is dangerous?

Reply:
We could pass fake value. I can add comment to avoid others worrying about the code.